apps/frontend/src/app/Database/DataManagers/CharacterData.ts

```diff
@@ -34,17 +34,17 @@ export class CharacterDataManager extends DataManager<CharacterKey, "characters"
 
     if (!allHitModes.includes(hitMode)) hitMode = "avgHit"
     if (reaction && !allAmpReactions.includes(reaction as typeof allAmpReactions[number]) && !allAdditiveReactions.includes(reaction as typeof allAdditiveReactions[number])) reaction = undefined
-    if (infusionAura!=="" && !allInfusionAuraElements.includes(infusionAura as InfusionAuraElements)) infusionAura = ""
+    if (infusionAura !== "" && !allInfusionAuraElements.includes(infusionAura as InfusionAuraElements)) infusionAura = ""
     if (typeof constellation !== "number" && constellation < 0 && constellation > 6) constellation = 0
 
     const { level, ascension } = validateLevelAsc(rawLevel, rawAscension)
 
     if (typeof talent !== "object") talent = { auto: 1, skill: 1, burst: 1 }
     else {
       let { auto, skill, burst } = talent
-      if (typeof auto !== "number" || auto < 1 || auto > 15) auto = 1
-      if (typeof skill !== "number" || skill < 1 || skill > 15) skill = 1
-      if (typeof burst !== "number" || burst < 1 || burst > 15) burst = 1
+      auto = typeof auto !== "number" ? 1 : clamp(auto, 1, 10)

```

Comment:
Should we clamp to 12 for skill+burst and 14 for auto? Or we can just add "Bonus Normal Talent Level" + etc to the bonus stats page in a different PR

Reply:
Its not selectable via the dropdown, which can cause problem with UI. the recommended way to add "bonuses" are from bonus stats like childe

Reply:
#926

Reply:
@nguyentvan7 is the expectation that I incorporate #926 here? I think changes here so far should be sufficient